lib/dependencies/ImportMetaContextDependencyParserPlugin.js

```diff
@@ -0,0 +1,98 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Ivan Kopeykin @vankop
+*/
+
+"use strict";
+
+const {
+	evaluateToIdentifier
+} = require("../javascript/JavascriptParserHelpers");
+const ImportMetaContextDependency = require("./ImportMetaContextDependency");
+
+/** @typedef {import("estree").Expression} ExpressionNode */
+/** @typedef {import("estree").ObjectExpression} ObjectExpressionNode */
+/** @typedef {import("../javascript/JavascriptParser")} JavascriptParser */
+
+/**
+ * @param {JavascriptParser} parser parser
+ * @param {ObjectExpressionNode} optionsNode node
+ * @returns {{mode: string, recursive: boolean, regExp: RegExp}} options
+ */
+function getOptions(parser, optionsNode) {
+	let regExp = /^\.\/.*$/;
+	let recursive = true;
+	let mode = "sync";
+	if (optionsNode) {
+		for (const prop of optionsNode.properties) {
+			if (prop.type !== "Property" || prop.key.type !== "Identifier") return;
+			switch (prop.key.name) {
```

Comment:
There are a bunch of other options we want to support here:

* `chunkName`
* `include`
* `exclude`
* `prefetch`
* `preload`
* `exports`